clients/admin-ui/src/features/common/PageHeader.tsx

```diff
@@ -1,4 +1,4 @@
-import { Box, BoxProps } from "fidesui";
+import { Box, BoxProps, Flex } from "fidesui";
```

Comment:
Added rightContent prop to PageHeader. Useful in new designs for buttons on the right of the page header.